include/golioth/client.h

```diff
@@ -231,23 +231,23 @@ enum golioth_status golioth_client_stop(struct golioth_client *client);
 /// @param client The handle of the client to destroy
 void golioth_client_destroy(struct golioth_client *client);
 
-/// Returns whether the client is currently running
+/// retvals whether the client is currently running
```

Comment:
find + replace going rogue 😉 
```suggestion
/// Returns whether the client is currently running
```

Reply:
:see_no_evil: